scripts/entrypoint.sh

```diff
@@ -173,6 +192,11 @@ if ! wait_for_ipam; then
     exit 1
 fi
 
+if ! wait_for_init; then
```

Comment:
I was a bit worried about the order here, as the init container sets some `sysctl`s for IPv4 and IPv6, but this should be fine as everything we set is for forwarding and no pods will be created until the config file move event. IPAMD checks env variable for IPv6, so this should be fine